### ISSUE TITLE
Add ability to edit plugin config from client extension

### ISSUE THREAD

**Issue by pinussilvestrus** [camunda/camunda-modeler#1512]
__Is your feature request related to a problem? Please describe.__

Make it possible for editor extension to set and get properties in the plugin config via a client extension. 

__Describe the solution you'd like__

Minimal API for accessing the already implemented Config implementation (cf. #1496) via client extension, so something like

```js
// plugin-component.js
const {
  config
} = props;


await config.setForPlugin(name, key, value);
```

__Additional context__

The overall setting and getting functionality was already implemented by #1494 

---

Child of #1496

**Comment by pinussilvestrus** [comment]
This will probably also help when it comes to implementing deployment configuration savings, cf. https://github.com/camunda/camunda-modeler/issues/1066

**Comment by pinussilvestrus** [comment]
Already exposed via https://github.com/camunda/camunda-modeler/commit/6abc68acf14918cfd334a247256ddffaed266f54, I will add a minimal example on how to use it in client extensions.

**Comment by pinussilvestrus** [comment]
The [`TestClient`](https://github.com/camunda/camunda-modeler/blob/develop/resources/plugins/test-client/client/TestClient.js) plugin offers a minimal example of how to use the config API in client extensions.

Closed via d868bdb38f9d573556a409126cff4c4c8ab8d2a4
